Utilities/Pipelines/libHttpClient.CI.yml

```diff
@@ -0,0 +1,168 @@
+# CI trigger. Fires on merges to specific branches.
+# Ref: https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git#ci-triggers
+# A pipeline with no CI trigger
+trigger: none
+
+# Nightly trigger. Builds on the specified schedule.
+# Refs:
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers#cron-syntax
+schedules:
+  # Schedule is specified in `cron` syntax. The below means: "8am UTC (midnight PST) every day".
+  - cron: "0 8 * * *"
+    displayName: Daily build
+    branches:
+      include:
+        - main
+    always: true
+
+# Name builds with the definition name, date, and build-count-for-that-day. For
+# example, "libHttpClient.CI_20210214.1".
+# Refs:
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/run-number
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables
+name: $(Build.DefinitionName)_$(date:yyMM).$(date:dd)$(rev:rrr)
+
+jobs:
+
+  ####################
+  # Visual Studio 2017
+  ####################
+
+  - job: Win32Vs17Build
+    displayName: libHttpClient Win32/UWP VS2017 Build
+    pool:
+      name: xbl-1es-vs2017
```

Comment:
Can these VS builds run on stock Windows agents, since they don't need XDK?

Reply:
This is a stock Windows agents owned by 1ES team (same image).  Microsoft hosted agents don't have scale or control options so aren't as good